polaris-react/src/components/ActionMenu/components/SecondaryAction/SecondaryAction.tsx

```diff
@@ -30,9 +32,11 @@ export function SecondaryAction({
   }, [getOffsetWidth]);
 
   const buttonMarkup = (
-    <Button onClick={onAction} {...rest}>
-      {children}
-    </Button>
+    <span ref={activatorRef}>
```

Comment:
[this](https://github.com/Shopify/polaris/pull/8701/files#diff-9c8f2ff3b7f4b56e6f5a156e5aa48458d62578bcbd45e8400ba0771760e32ece) resolves having to wrap buttons to include refs